junit4assignment/src/test/java/org/example/StringUtilsTest.java

```diff
@@ -0,0 +1,69 @@
+package org.example;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.*;
+public class StringUtilsTest
+{
+    StringUtils stringUtils;
+    @Before
+    public void setup()
+    {
+        stringUtils = new StringUtils();
+    }
+
+    // testing reverse method with non empty string.
+    @Test
+    public void testReverseShouldReturnReverse_test1()
```

Comment:
_test1 _test2 , why all methods with these suffixes 

testReverse_givenExpectedInput -- like this, we need to have the name that reflects scenario as suffix.